controllers/user/views/forgotten-password.njk

```diff
@@ -21,18 +21,22 @@
                 {% if csrfToken %}
                     <input type="hidden" name="_csrf" value="{{ csrfToken }}">
                 {% endif %}
+                
 
                 {{ formField({
                     type: 'email',
                     name: 'username',
                     label: 'Email address',
+                    explanation: 'Enter the email address associated with your account.',
                     isRequired: true,
                     value: formValues['username']
                 }, errors = errors) }}
 
                 <div class="form-actions">
-                    <input type="submit" class="btn" value="Reset password request" />
+                    <input type="submit" class="btn" value="Request password reset" />
```

Comment:
🙃

Reply:
Honestly not sure If this more or less grammatically correct now. We might be better off with simply "Reset password: